CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -421,7 +421,7 @@ CF_PRIVATE Boolean __CFProcessIsRestricted(void);
 
 CF_EXPORT void * __CFConstantStringClassReferencePtr;
 
-#if DEPLOYMENT_RUNTIME_SWIFT
+#if DEPLOYMENT_RUNTIME_SWIFT && TARGET_OS_MAC
 
 #if DEPLOYMENT_TARGET_LINUX
 #define CONST_STRING_SECTION __attribute__((section(".cfstr.data")))
```

Comment:
Is this still used?  The section itself is not compatible with what clang has done for a very long time.

Reply:
Yeah; this is the path used when clang is _not_ involved in laying out the CFSTR(). We still need it for Mac as outlined in the PR description.

Reply:
Should the section be updated to be correct then?  The literal is placed into `.rodata.str1.1`, the structures into `cfstring`.

Reply:
I'll just remove the dead code.

Reply:
Per discussion: this seems to be diff'd against an older revision than master, and that code is no longer in apple/swift-corelibs-foundation.